screen.html

```diff
@@ -0,0 +1,389 @@
+<html>
```

Comment:
I would need a demo, my simulator is somewhat broken. We would need to have a look: https://www.airconsole.com/simulator/#http://0.0.0.0:8091/

Reply:
While serving API repo on 8091 of course

Reply:
that will not work. https + http gives you a broken infinite loop thanks to Chrome / Safari trying to downgrade and then getting upgraded again and again.

Using http://http.airconsole.com/simulator/#http://0.0.0.0:8091/ should work.
Not sure how we can deal with this in pure web going forward. In Unity v2.14 is taking care of this protocol split brain problem